src/pages/ListPage/CreatePolicyWizard.tsx

```diff
@@ -80,9 +81,10 @@ export const CreatePolicyWizard: React.FunctionComponent<CreatePolicyWizardProps
                 onClose={ () => { props.close(false); } }
                 onSave={ onSave }
                 onVerify={ onVerify }
-                showCreateStep={ props.showCreateStep }
+                showCreateStep={ props.policiesExist ? props.showCreateStep : false }
                 isLoading={ isLoading }
-            /> }
+                policiesExist={ props.policiesExist }
+            /> } 
```

Comment:
Travis is showing a linter error: `/home/travis/build/RedHatInsights/policies-ui-frontend/src/pages/ListPage/CreatePolicyWizard.tsx
  87:17  error  Trailing spaces not allowed  no-trailing-spaces`
```suggestion
            />} 
```